backtester/eventhandlers/exchange/exchange.go

```diff
@@ -406,12 +417,17 @@ func (e *Exchange) placeOrder(ctx context.Context, price, amount decimal.Decimal
 }
 
 func applySlippageToPrice(direction gctorder.Side, price, slippageRate decimal.Decimal) decimal.Decimal {
-	adjustedPrice := price
-	if direction == gctorder.Buy {
+	var adjustedPrice decimal.Decimal
+	switch direction {
+	case gctorder.Buy, gctorder.Bid, gctorder.Long:
```

Comment:
Can do side method direction.IsLong() and isShort() also can default error if you so choose.

Reply:
As in, you'd like a function in the orderpackage `IsLong()` which checks if asset == Long?